### PR TITLE
Removes usages of ExperimentActivity.

### PR DESCRIPTION
**Pull request by ashercodeorg**
Another PR will perform the migration to drop the DB table.

### CODE REVIEW

File: dashboard/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -179,22 +179,6 @@ def milestone_response(options)
...
-                                                         enable_external_hints: Rails.env.production?,
-                                                         ip: request.remote_ip,
-                                                         uri: request.referer,
-                                                         activity: options[:activity]})
-      response[:hint] = hint_details[:hint] if hint_details[:hint]
-      response[:hint_request_placement] = hint_details[:hint_request_placement] if
-        hint_details[:hint_request_placement]
-      response[:hint_requested_url] =
-        activity_hint_path(hint_details[:activity_hint]) if hint_details[:activity_hint]
-    end
```
Comment (ashercodeorg) [inline dashboard/app/controllers/application_controller.rb:196]: Though I have done a best effort confirming that these are never referenced, this should be confirmed rather carefully.

**Comment by Hamms** [comment]
Can I assume adding a migration to remove the created table will be in another PR, after we've had a chance to dump the existing data?

**Comment by ashercodeorg** [comment]
Yes (PR is #10128). And data (just over 1 GB) has already been backed up to AWS s3.

File: dashboard/app/controllers/application_controller.rb (hunk 1 above)
Comment (Hamms) [inline dashboard/app/controllers/application_controller.rb:196]: I believe `response[:hint]` is referenced [in feedbackBlocks](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/feedbackBlocks.js#L10) and a couple places [in feedback](https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/feedback.js#L438).

**Comment by Hamms** [comment]
LGTM. It would be nice to remove some of that JS we no longer need, but it seems reasonable to put that in a separate PR.

**Comment by ashercodeorg** [comment]
PTAL. The dead JS code has been purged.

**Comment by Hamms** [comment]
wooooo! LGTM!

**Pull request merged**